quasar-cli-vue3-webpack-javascript/app-exposes/.quasar/client-entry.js

```diff
@@ -49,7 +49,10 @@ console.info('[Quasar] Running SPA.')
 
 
 
-const publicPath = ``
+const publicPath = `http://localhost:3001/`
+
+const doubleSlashRE = /\/\//
+const addPublicPath = url => (publicPath + url).replace(doubleSlashRE, '/')
```

Comment:
Just asking shall we use global replace/replace all occurrences here?

Reply:
I believe it's autogenerated folder